### PR TITLE
Social Distance Checker

### PR DESCRIPTION
**Pull request by rahulgrandhi13579**

### CODE REVIEW

**Review by theobscuredev — CHANGES_REQUESTED** [review]
Hey @rahulgrandhi13579 

Few changes are required!!!!

1. Update the README file. Refer the Documentation Guidelines for the same. While you prepare the README, make sure you consider the image aligning and text justification using appropriate tags. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md
2. Update the name of the 'Poster and Report'. 'Images and Videos', 'Images' etc folders. Make sure all the directories are named as per the convention mentioned in the guidelines.
3. Also, update the name of all the files. Avoid naming them as 'IMG_20210806_221926.jpg'. Naming should be self-explanatory and should justify the content present within.
4. Update the project report. Add a section for mentors and elaborate a few slides.
5. Try to have a uniform naming convention for the files. Either use CamelCase (For eg: CentroidTracker.py or centroidTracker.py)/ use underscores (centroid_tracker.py)/ use hyphens (centroid-tracker.py) but make sure it's uniform throughout the entire codebase.

**Comment by theobscuredev** [comment]
@rahulgrandhi13579 Lemme know once you've made the final changes. Make sure to tag me (@) in this thread.

**Comment by rahulgrandhi13579** [comment]
Sir, we have made the necessary changes that you have stated . So once
could you please check them whether they are correct or do we need to do
any other changes.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
 # Social-Distance-Checker
 Open Projects 2021
```
Comment (theobscuredev) [inline README.md]: Apply header using: '## Open Projects 2021'

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
 # Social-Distance-Checker
```
Comment (theobscuredev) [inline README.md]: ~~Social-Distance-Checker~~ -> Social Distance Checker

File: README.md [hunk 3]
```diff
@@ -1,2 +1,65 @@
...
 Open Projects 2021
+
+
+![socialdistancing_mask_detection](https://user-images.githubusercontent.com/88554453/128603773-dceafea5-7f4f-46a8-8303-98dc23235368.jpg)
```
Comment (theobscuredev) [inline README.md]: Center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `LED Matrix`
  `<i>LED Matrix</i>`
`</p>`

Also resize the images by assigning the dimensions and add title for each pic. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

File: README.md [hunk 4]
```diff
@@ -1,2 +1,65 @@
...
+
+# Motivation
+
+The Outbreak of COVID-19 has affected almost everyone across the globe. During this pandemic, people are 
+advised not to be in close contact with others to reduce the spread of the disease, but due to the negligence 
+of many, we continue to face the repercussions of this pandemic
+As the name suggests, social distancing implies that people should physically distance themselves from 
+one another, reducing the spread of contagious diseases like COVID. So, in order to monitor if people are 
+maintaining social distance or not, we are working on the project named, Social distance checker. 
+
```
Comment (theobscuredev) [inline README.md:32]: Insert a 'Software Aspect' heading and elaborate your algorithm. If it's taking up too much space or if it's too elaborate, you can also create a separate README for the same. For eg: https://github.com/brahatesh/RoverSim#software-aspects
Make sure you're able to explain the logic of the algorithm in the context of the code written. Flowcharts or code snippets can be used for explanation.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,65 @@
...
+# Application
+
+It is used to maintain social distancing and helps in preventing the spread of deadly communicable diseases.
```
Comment (theobscuredev) [inline README.md]: Elaborate your project applications. Consider all the aspects like how exactly people can use the features in your app to 'Prevent the spread of the virus'.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,65 @@
...
+
+# Limitations
+
+1.) It fails to detect the persons when the video of one of person gets blocked by another person.
```
Comment (theobscuredev) [inline README.md]: Use proper bullet points. Remove the unnecessary fullstop after each digit.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,65 @@
...
+
+5.) Vansh Mehra
+
+# Mentors
+
+1.) Agrim Aggarwal
+
+2.) Anfal Ansari
+
+3.) Annushree
```
Comment (theobscuredev) [inline README.md]: Hyperlink the Github ID of each team member & your mentors. Refer this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

File: README.md [hunk 8]
```diff
@@ -1,2 +1,65 @@
...
+
+# References 
+
+1.) https://youtu.be/UM9oDhhAg88
+
+2.) https://youtu.be/Ax6P93r32KU
```
Comment (theobscuredev) [inline README.md]: Hyperlink the references properly. Mention the title for each reference and just include the URL in the hyperlink. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

File: README.md [hunk 9]
```diff
@@ -1,2 +1,65 @@
 # Social-Distance-Checker
 Open Projects 2021
+
+
+![socialdistancing_mask_detection](https://user-images.githubusercontent.com/88554453/128603773-dceafea5-7f4f-46a8-8303-98dc23235368.jpg)
+
+# Abstract 
+
+The social distance checker is an opencv project in which we monitor the distances between the people and if the distance comes out to be less than the threshold value (calibrated six feet in terms of pixels) then we indicate the violation by surrounding them with a red bounding box and with the help of AI/ML/pretrained files , if the person does not wear a mask then his face gets surrounded by a red bounding box.
```
Comment (theobscuredev) [inline README.md]: Justify the entire text using the `<p>` paragraph tag and `justify` attribute. Make sure you justify all the paragraphs separately. For eg:
`<p align="justify">`
  `<h2>Abstract</h2>`
  `<p>A Rube Goldberg Machine is .........................</p>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

File: README.md [hunk 10]
```diff
@@ -1,2 +1,65 @@
...
+# Motivation
+
+The Outbreak of COVID-19 has affected almost everyone across the globe. During this pandemic, people are 
+advised not to be in close contact with others to reduce the spread of the disease, but due to the negligence 
+of many, we continue to face the repercussions of this pandemic
+As the name suggests, social distancing implies that people should physically distance themselves from 
+one another, reducing the spread of contagious diseases like COVID. So, in order to monitor if people are 
+maintaining social distance or not, we are working on the project named, Social distance checker. 
+
+# Application
```
Comment (theobscuredev) [inline README.md]: Include a tabulated cost structure depicting the price of any paid software used before the Applications. For eg: https://github.com/marsiitr/IoT-ChatBot#cost-structure

**Review by theobscuredev — CHANGES_REQUESTED** [review]
Hey @rahulgrandhi13579 & team;
Nice work however lots of changes still have to be made:

1. Capitalize 'i' and 'v' while naming the 'Images and Videos', 'Images' and 'Videos' directories. 
2. Update the README file as per the changes suggested. Try to elaborate your algorithm using code snippets/flowcharts. For eg: https://github.com/brahatesh/RoverSim#software-aspects. If need arises, you can create a new README and include it inside the src folder. Make sure others can understand the code logic by referring the README file which isn't clear currently.
3. Insert more images in the README and center align them after adding a corresponding title.

**Comment by rahulgrandhi13579** [comment]
Sir, we have made all the necessary changes that you have stated . Are
these correct or need any further changes?

**Review by SanjeevKrishnan — APPROVED** [review]
Good work team!
Merging the PR

**Pull request merged**